packages/client/src/components/PlaceholderPages/ContactUs/ContactUs.styles.css

```diff
@@ -0,0 +1,166 @@
+* {
+  background-color: #e4c1b1;
+  margin: 0 auto;
+}
+.contactUs-wrapper {
+  display: flex;
+  justify-content: center;
+}
+.contact-info-container {
+  border-radius: 35px;
+  width: 672px;
+  height: 982px;
+  background: #f2ebe7;
+}
+.contact-info {
+  display: inline-block;
+  margin-top: 118px;
+  margin-right: 96px;
+  background: #f2ebe7;
+}
+.contact-info h3 {
+  font-family: Montserrat;
+  font-size: 24px;
+  text-align: left;
+  line-height: 29px;
+  margin-bottom: 25px;
+  background: #f2ebe7;
+}
+.fashion-email {
+  display: flex;
+  margin-bottom: 35px;
+  border: 1px solid #f5f5f5;
+  border-radius: 8px;
+  background: #f5f5f5;
+}
+.fashion-email img {
+  width: 35px;
+  height: 35px;
+  background: #f5f5f5;
+  cursor: pointer;
+  margin-left: 10px;
+}
+.fashion-email p {
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 29px;
+  color: #212121;
+  background: #f5f5f5;
+  margin-left: 10px;
+}
+.or {
+  margin-bottom: 30px;
+  background: #f2ebe7;
+}
+form {
+  display: flex;
+  flex-direction: column;
+  background: #f2ebe7;
+}
+.contactUs-input {
+  border-style: hidden;
+  border-bottom: 1px solid rgba(33, 33, 33, 0.4);
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  width: 408px;
+  background: #f2ebe7;
+  text-align: start;
+  color: #212121;
+}
+.contactUs-input::placeholder {
+  padding: 0;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 15px;
+  font-family: 'Montserrat';
+}
+.contactUs-btn {
+  width: 408px;
+  height: 45px;
+  background: #f5f5f5;
+  border: 1px solid #f5f5f5;
+  border-radius: 8px;
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 29px;
+  margin-top: 35px;
+  margin-bottom: 55px;
+}
+
+.media-icons {
+  display: flex;
+  background: #f2ebe7;
+}
+.media-icons img {
+  width: 36px;
+  height: 36px;
+  cursor: pointer;
+  margin-right: 23px;
+  background: #f2ebe7;
+}
+
+@media screen and (max-width: 1024px) {
+  .contact-info-container {
+    width: 370px;
+    height: 782px;
+  }
+  .contactUs-img {
+    width: 350px;
+    height: 782px;
+  }
+  .contact-info {
+    margin-left: 8px;
+  }
+  .contactUs-btn,
+  .contactUs-input,
+  .fashion-email {
+    width: 350px;
+  }
+  .contact-info h3 {
+    font-size: 20px;
+  }
+  .fashion-email p,
+  p,
+  .contactUs-btn {
+    font-size: 15px;
+  }
+}
+@media screen and (max-width: 480px) {
+  .contact-info-container {
```

Comment:
If this is designed for max 480px screen, but the image and the form together is more than that, the page becomes horizontally scrollable, which is not good 😿 
You can make a jira and a TODO to fix this later.

Reply:
alright I can fix that now